src/ui/App.swift

```diff
@@ -178,7 +178,7 @@ class App: AppCenterApplication, NSApplicationDelegate {
 
     func showSecondaryWindow(_ window: NSWindow?) {
         if let window = window {
-            NSScreen.preferred().repositionPanel(window, .appleCentered)
+            NSScreen.preferred().repositionPanel(window, .centered)
```

Comment:
Once we are firm on the theme and appearance tab, I will ensure all the static code is dynamic.